tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -240,7 +240,8 @@ public:
       NTupleSize_t fNElements = kInvalidNTupleIndex;
       /// The usual format for ROOT compression settings (see Compression.h).
       /// The pages of a particular column in a particular cluster are all compressed with the same settings.
-      int fCompressionSettings = kNTupleUnknownCompression;
+      /// If unset, the compression settings are undefined (deferred columns, suppressed columns).
+      std::optional<std::uint32_t> fCompressionSettings;
```

Comment:
Does it change anything to the onfiles files in the columns meta data?

Reply:
No, there is no impact on the on disk data.